### PR TITLE
Use events fully for layout css and js

### PR DESCRIPTION
**Pull request by Prometee**
| Q               | A
| --------------- | -----
| Branch?         | 1.7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | none
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml [hunk 1]
```diff
@@ -93,15 +93,23 @@ sylius_ui:
         
         sylius.admin.index.stylesheets:
             blocks:
+                styles:
+                    template: "@SyliusAdmin/Layout/_styles.html.twig"
+                    priority: 10
                 legacy:
                     template: "@SyliusAdmin/Crud/Block/_legacySonataEvent.html.twig"
+                    priority: 0
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml]:
```suggestion
                    priority: 5
```

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml [hunk 2]
```diff
@@ -93,15 +93,23 @@ sylius_ui:
...
                         postfix: index.stylesheets
                         
         sylius.admin.index.javascripts:
             blocks:
+                scripts:
+                    template: "@SyliusAdmin/Layout/_scripts.html.twig"
+                    priority: 10
                 legacy:
                     template: "@SyliusAdmin/Crud/Block/_legacySonataEvent.html.twig"
+                    priority: 0
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml]:
```suggestion
                    priority: 5
```

**Comment by Prometee** [comment]
@lchrusciel I will change those priority and use the right event too. I commit the wrong event with `sylius.admin.index.*`, the right one is `sylius.admin.layout.*`

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml [hunk 3]
```diff
@@ -237,15 +239,23 @@ sylius_ui:
...
                 legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 5
                     context:
                         event: sylius.admin.layout.javascripts
                         
         sylius.admin.layout.stylesheets:
             blocks:
+                styles:
+                    template: "@SyliusAdmin/Layout/_styles.html.twig"
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml]: Is the file name proper? You've deleted `{% include '@SyliusAdmin/_styles.html.twig' %}`

Comment (Prometee) [reply src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml]: No I did not delete it, but it has been mark as deprecated so I guess I have to set `@SyliusAdmin/_styles.html.twig` now and wait for deletion of the deprecated file ?

**Comment by lchrusciel** [comment]
Thank you, Francis! :1st_place_medal:

**Pull request merged**